packages/repo-app-import-sub-generator/src/prompts/prompts.ts

```diff
@@ -86,21 +122,27 @@ export async function getPrompts(
                 },
                 type: 'list',
                 name: PromptNames.selectedApp,
-                default: () => (quickDeployedAppConfig?.appId ? 0 : undefined),
+                default: (): number | undefined => (quickDeployedAppConfig?.appId ? 0 : undefined),
                 guiOptions: {
                     mandatory: !!appList.length,
                     breadcrumb: t('prompts.appSelection.breadcrumb'),
                     applyDefaultWhenDirty: true
                 },
                 message: t('prompts.appSelection.message'),
                 choices: (): { name: string; value: AppInfo }[] => (appList.length ? formatAppChoices(appList) : []),
-                validate: async (answers: AppInfo) =>
-                    validateAppSelection(answers, appList, quickDeployedAppConfig, appWizard)
+                validate: async (answers: AppInfo): Promise<boolean> => {
+                    const result = await validateAppSelection(answers, appList, quickDeployedAppConfig, appWizard);
```

Comment:
Same comment, as long as `validateAppSelection` handles undefined for `answers` should be OK?

Reply:
yeah it handles it [https://github.com/SAP/open-ux-tools/blob/05890cec71371d9a8bd9f30f45909ba200695ce9/packages/repo-app-import-sub-generator/src/utils/validators.ts#L143](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html)